### PR TITLE
docs(ui) : updated readme link

### LINKED ISSUE #45: [DOCS] Fix API and UI setup links in global readme
Currently `ui setup` and `api setup` both point to `ui` README.

Action Items:
- [ ]  Update the `api setup` link to point to API's README.
- [ ]  Clicking on the link should land and scroll to the README instead of at the top of the screen.

### PR DESCRIPTION
**Pull request by PrajwalBorkar**
docs(ui) : updated readme link
Fixes #45

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ merchandise, etc. for all the gymkhana clubs in our institute.
 - Fork and clone the repository.
 - Add remote upstream `git add upstream https://github.com/BitByte-TPC/gymkhana.git`
 - [ui setup](https://github.com/BitByte-TPC/gymkhana/tree/master/ui)
```
Comment (Samy-33) [inline README.md:18]: please add readme # link to ui as well

**Review by Samy-33 — CHANGES_REQUESTED** [review]
1. Get the issue #45 assigned to you before you start working on the change.
2. Update your commit message to conform to the [guidelines](https://github.com/BitByte-TPC/gymkhana/blob/master/CONTRIBUTING.md#standard-commit-messages)

**Comment by PrajwalBorkar** [comment]
> 1. Get the issue #45 assigned to you before you start working on the change.
> 2. Update your commit message to conform to the [guidelines](https://github.com/BitByte-TPC/gymkhana/blob/master/CONTRIBUTING.md#standard-commit-messages)

Made the changes please review.

**Comment by Samy-33** [comment]
1. You've still not gotten the issue assigned.
2. Please squash your commits into one. And change the commit message.

**Comment by PrajwalBorkar** [comment]
> 1. You've still not gotten the issue assigned.
> 2. Please squash your commits into one. And change the commit message.

Should I close the PR if issue is not assigned to me?

**Comment by Samy-33** [comment]
> Should I close the PR if issue is not assigned to me?

The issue is still not assigned to anyone so you can get it assigned to yourself.
Just comment on the issue that you'll be working on it. I will assign it to you.

**Pull request closed without merging**